packages-next/keystone/src/session/index.ts

```diff
@@ -115,7 +115,7 @@ export function withItemData<T extends { listKey: string; itemId: string }>(
   };
 }
 
-export function statelessSessions<T>({
+export function statelessSessions<T extends Record<string, any>>({
```

Comment:
This restriction seems unnecessary. Iron supports anything that can be json stringified.

Reply:
Right, but `withItemData` can't use it unless it's an object.

Reply:
Sure withItemData requires an object, why should statelessSessions require it though?